docs/concepts/configuration/assign-pod-node.md

```diff
@@ -187,7 +187,7 @@ value V that is running a pod that has a label with key "security" and value "S1
 rule says that the pod prefers to not schedule onto a node if that node is already running a pod with label
 having key "security" and value "S2". (If the `topologyKey` were `failure-domain.beta.kubernetes.io/zone` then
 it would mean that the pod cannot schedule onto a node if that node is in the same zone as a pod with
-label having key "security" and value "S2".) See the [design doc](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md).
+label having key "security" and value "S2".) See the [design doc](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md) 
```

Comment:
Replace this line with:
```
label having key "security" and value "S2".) For more information, see the [design doc](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md).
```
So:
> label having key "security" and value "S2".) For more information, see the [design doc](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md).

Reply:
@zacharysarah : I only removed the dot, it should not be there, because the next line is `for many more examples of pod affinity and anti-affinity`, it has the meaning that you have suggested.This line should be in conjunction with line 191.

Reply:
@zacharysarah I think liubin has the right answer here - going to approve through